.gitignore

```diff
@@ -1,6 +1,5 @@
 /out
 /build
 /target
-/.idea
```

Comment:
What the benefits of sharing `.idea` folder?

Reply:
Recommendation by the IntelliJ IDEA team. And if I'm not mistaken the shared indexes reside here so that it makes things faster?